### ISSUE TITLE
Maintain Scroll Position (Fails on Mobile)

### ISSUE THREAD

**Issue by zgjonbalaj** [liabru/jquery-match-height#121]
I am having an issue with the maintain height on mobile phone, the page will jump to the top when you scroll all the way down. The flag is there but it doesnt seem to do any good, ive tried a few different things including a timeout, data-mh attributes.

This is my markup

`div class="col-xs-12 col-md-5 col-lg-5">
				<div class="left-footer-col eh clearfix">
					<address>
						<span class="company">Company Name</span>
						<span class="address">Company Address</span>
						<span class="phone">Phone: (555) 123-4567 | </span>
						<span class="fax">Fax: (555) 123-4567</span>
					</address>

					<div class="copyright">
						&reg; Copyright <?= date('Y') ?> Company Name. <span class="no-wrap">All rights reserved.</span>
					</div>
				</div>
			</div>

			<div class="col-xs-12 col-md-7 col-lg-7">
				<div class="right-footer-col eh clearfix">
					<? require("_includes/nav/footer.inc") ?>
				</div>
			</div>`

This is my JS

`
$(document).ready(function() {
  $.fn.matchHeight._maintainScroll = true;
});

$(window).load(function() {
  // MatchHeight .eh Elements
  setTimeout(function() {
    $(".eh").matchHeight();
  }, 1);
});

$(window).resize(function() {
  // MatchHeight Update
  $.fn.matchHeight._update()
});

`

**Comment by liabru** [comment]
Can you tell me the model of the phone and the browser version?

**Comment by zgjonbalaj** [comment]
It seems to be on all phones we've tested including

1. Android Nexus 6 - Running Android 6.0.1
2. iPhone 5 & 6

*Zgjim (Jim) Gjonbalaj, Owner/Web Developer*
*Computer City Service L.L.C*
*computercityservice.com <http://computercityservice.com> |
info@computercityservice.com <info@computercityservice.com>*
*97 Myrtle Ave Ansonia, CT 06401*
*Office: (203) 516-0505 | Cell: (203) 893-3714*

**Comment by danjaywing** [comment]
Did this ever get resolved? I'm noticing this on ios, page jumps caused by the _update method, _maintainScroll is set to true

**Comment by zgjonbalaj** [comment]
Nope, still experience this quite often my only fix is to add a set time
out delay to the function. Moving it around to page load instead of
document ready also seems to help.

**Comment by liabru** [comment]
Unfortunately `_maintainScroll` is only approximate, certain content changes will throw the estimation off more than others. Maybe there's a bug with newer mobile browsers, does anybody have a good example link showing it?
